### DISCUSSION TITLE
Behind the Scenes: Improving Project Structure and Other Updates

### DISCUSSION THREAD

**Discussion by PrestonN** [FreeTubeApp/FreeTube#1780] [Blog Post]
https://blog.freetubeapp.io/behind-the-scenes-improving-project-structure-and-other-updates

**Comment by mdlos** [comment]
Hello, I'm really enjoying the application and I've been using it on my manjaro linux. One suggestion, could be made an implement to stream the video stream via chromecast.
Congratulations on freetube.

**Comment by ckolumbus** [comment]
Great work! Thanks for this piece of Software. I'm glad to see that a team has of contributors help to keep up the maintenance of such a large project! 👍🏽
